components/combobox/__examples__/input-prop.jsx

```diff
@@ -79,8 +79,8 @@ class Example extends React.Component {
 		// example of passing in an SLDSInput component to a SLDSCombobox
 		const inputElement = (
 			<Input
+				label="Search"
 				autoComplete="test"
-				id="id_from_input_prop"
```

Comment:
@interactivellama why was this id removed? not needed for snapshot testing?

Reply:
Yes, it was not needed.